src/components/form/PrintPDF.js

```diff
@@ -99,6 +102,13 @@ export default class PrintPDF extends  React.Component
       doc.setFontSize(14).text(438, 210, physical_defense + '', 'center');
     }
 
+    //recoveries
+    var recoveryDescription = doc.setFontSize(8).splitTextToSize(recovery_description, 90)
+
+    doc.setFontSize(10).text(170, 185, recoveries, 'center');
```

Comment:
Can this and line 109 be standardized at all? Can those x and y values be pulled from our object for all constant values? Since this is such as simple one it's fine if not, but just something to consider.